UnityProject/Assets/Scripts/Items/Implants/Organs/Heart.cs

```diff
@@ -62,11 +62,13 @@ public override void ImplantPeriodicUpdate()
 
 		public override void RemovedFromBody(LivingHealthMasterBase livingHealth)
 		{
+			//livingHealth.reagentPoolSystem.PumpingDevices.Remove(this);
```

Comment:
remove instead of commneting out

Reply:
it's just commented out so I can swap back to the new system